service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandler.java

```diff
@@ -36,20 +37,21 @@
     private final CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
     private final CafcassLookupConfiguration cafcassLookupConfiguration;
     private final IssuedOrderAdminNotificationHandler issuedOrderAdminNotificationHandler;
-    private final DocumentDownloadService documentDownloadService;
+
+    @Value("${manage-case.ui.base.url}")
+    private String xuiBaseUrl;
 
     @EventListener
     public void notifyParties(final CaseManagementOrderIssuedEvent event) {
         CaseData caseData = event.getCaseData();
         CaseManagementOrder issuedCmo = event.getCmo();
 
-        //TODO Document is downloaded 5 times, this could impact performance. Investigate in FPLA-2061
         sendToLocalAuthority(caseData, issuedCmo);
         sendToCafcass(caseData, issuedCmo);
         sendToRepresentatives(caseData, issuedCmo, DIGITAL_SERVICE);
         sendToRepresentatives(caseData, issuedCmo, EMAIL);
         issuedOrderAdminNotificationHandler.notifyAdmin(caseData,
-            documentDownloadService.downloadDocument(issuedCmo.getOrder().getBinaryUrl()), CMO);
+            concatUrlAndMostRecentUploadedDocumentPath(xuiBaseUrl, issuedCmo.getOrder().getBinaryUrl()), CMO);
```

Comment:
technically this changes how things work currently look for admin (but they should be seeing the link to document in case, not the download - needs business to verify)

Reply:
business said that's correct, so I will also make the change for C110a for admin